dpe/src/commands/derive_context.rs

```diff
@@ -392,11 +473,11 @@ mod tests {
         };
         let mut dpe = DpeInstance::new(&mut env, Support::AUTO_INIT).unwrap();
 
-        DeriveChildCmd {
+        DeriveContextCmd {
             handle: ContextHandle::default(),
             data: [0; DPE_PROFILE.get_tci_size()],
-            flags: DeriveChildFlags::MAKE_DEFAULT | DeriveChildFlags::CHANGE_LOCALITY,
-            tci_type: 7,
+            flags: DeriveContextFlags::MAKE_DEFAULT | DeriveContextFlags::CHANGE_LOCALITY,
+            tci_type: 0,
```

Comment:
nit: not a big deal, but why change this?

Reply:
It's because I added the check `dpe.contexts[parent_idx].tci.tci_type != self.tci_type`. I see now that this is only for the recursive case so changing it back.